src/containers/Home/index.js

```diff
@@ -0,0 +1,18 @@
+// @flow
+
+import React, {Component} from 'react';
+import {Jumbotron} from 'react-bootstrap';
+
+export class Home extends Component<void> {
```

Comment:
when components don't have an internal state there is no need for them to be a class and they should be coded as pure functions. so home will become:

```javascript

// @flow

import React, {Component} from 'react';
import {Jumbotron} from 'react-bootstrap';

const Home = () => <Jumbotron>
    <p className="App-intro">
      To get started, edit <code>src/App.js</code> and save to reload.
    </p>
</Jumbotron>

export default Home;

```

Reply:
This component does not have a state, but since this is a boilerplate, it could need one when you add functionality to it. That's why I left it as a class instead of a function.

Reply:
If that happens then it will become a container component that will render the Home Component and pass props to it. So this one will still be a function.

Plus leaving things in an invalid state hoping for a future change is not a valid reason IMO.